kudo-operator/templates/crds.yaml

```diff
@@ -0,0 +1,5091 @@
+                                      type: object
+                                    type: array
+                                  path:
+                                    type: string
+                                  port:
+                                    anyOf:
+                                      - type: string
+                                      - type: integer
+                                  scheme:
+                                    type: string
+                                required:
+                                  - port
+                                type: object
+                              tcpSocket:
+                                properties:
+                                  host:
+                                    type: string
+                                  port:
+                                    anyOf:
+                                      - type: string
+                                      - type: integer
+                                required:
+                                  - port
+                                type: object
+                            type: object
+                          preStop:
+                            properties:
+                              exec:
+                                properties:
+                                  command:
+                                    items:
+                                      type: string
+                                    type: array
+                                type: object
+                              httpGet:
+                                properties:
+                                  host:
+                                    type: string
+                                  httpHeaders:
+                                    items:
+                                      properties:
+                                        name:
+                                          type: string
+                                        value:
+                                          type: string
+                                      required:
+                                        - name
+                                        - value
+                                      type: object
+                                    type: array
+                                  path:
+                                    type: string
+                                  port:
+                                    anyOf:
+                                      - type: string
+                                      - type: integer
+                                  scheme:
+                                    type: string
+                                required:
+                                  - port
+                                type: object
+                              tcpSocket:
+                                properties:
+                                  host:
+                
```

Comment:
should we break these two crds up into their own files so its easier to know which file to edit if we ever need to? Or is it if we ever change one value for `sparkapplications` we'll likely change it for `scheduledsparkapplications` too?

Reply:
These CRDs shouldn't be ever edited manually.  We are discussing with @akirillov approach how to maintain the yaml. In general, it should be auto-generated like described [here](https://github.com/mesosphere/spark-on-k8s-operator/blob/master/docs/developer-guide.md)

Reply:
ah, autogenerated. okay thanks